theme/theme.go

```diff
@@ -545,6 +545,98 @@ func PrimaryColorNamed(name string) color.Color {
 	return col
 }
 
+//ColorToNRGBA converts a color to RGBA values which are not premultiplied, unlike color.RGBA().
+func ColorToNRGBA(c color.Color) (r, g, b, a int) {
```

Comment:
I'm not sure this warrants a new public API - perhaps it could be somewhere in the `internal/` space?

Reply:
Is there anywhere in particular in internal/ that you would like it to go? Also, unmultiplyAlpha and Test_UnmultiplyAlpha will have to go with it, of course.

Reply:
Possibly an `internal/color.go` file? we need to tidy that area up a little so other suggestions welcome.

Reply:
Since `internal/color.go` results in an import cycle error, how about `internal/col/color.go`? That way there's no possibility of one. Alternately, `internal/color/color.go`, since 'col' is not a super obvious name, in which case everything importing it would have to import it like `import col internal/color` because they all also import `image/color`.

Reply:
I've put it in internal/color/color.go since 'col' could mean anything.